### DISCUSSION TITLE
Will step names support replacements?

### DISCUSSION THREAD

**Discussion by ZacSweers** [community/community#25864] [Actions]
<p>Right now if you have a <code>jobs.\&lt;job-id\&gt;.steps.name</code> with a <code>Configure Gradle ${{ matrix.java_version }}</code> in it, the name comes out mangled. This does work fine with jobs.&lt;job-id&gt;.name though.</p>
<p>Example:  </p><div class="lightbox-wrapper"><a class="lightbox" href="https://aws1.discourse-cdn.com/github/original/2X/9/9d0b92394c9305f119c2b8180ec0c47329877781.png" data-download-href="/uploads/short-url/mphKKqCZl5Gk3S5ttrZAKTwgCEp.png?dl=1" title="image.png" rel="nofollow noopener"><img src="https://user-images.githubusercontent.com/1361086/181093766-957a1d91-1b6d-4708-8305-e5472b41cf36.png" alt="image.png" data-base62-sha1="mphKKqCZl5Gk3S5ttrZAKTwgCEp" width="690" height="77" data-small-upload="https://aws1.discourse-cdn.com/github/optimized/2X/9/9d0b92394c9305f119c2b8180ec0c47329877781_2_10x10.png"><div class="meta">
<svg class="fa d-icon d-icon-far-image svg-icon" aria-hidden="true"><use xlink:href="#far-image"></use></svg><span class="filename">image.png</span><span class="informations">872×98 5.8 KB</span><svg class="fa d-icon d-icon-discourse-expand svg-icon" aria-hidden="true"><use xlink:href="#discourse-expand"></use></svg>
</div></a></div>

**Comment by hugovk** [comment]
<p>+1, see also <a href="https://github.com/actions/setup-python/issues/19" rel="nofollow noopener">https://github.com/actions/setup-python/issues/19</a></p>

**Comment by ethomson** [comment, marked as answer]
<p>They don’t yet support them, but they will.  Apologies for the inconvenience in the meantime.</p>

**Comment by hugovk** [comment]
<p>This is now working:</p>
<p><img src="https://user-images.githubusercontent.com/1324225/66550022-31154f00-eb4d-11e9-90a4-7a3866d6a476.png" alt=""></p>
<p><a href="https://github.com/hugovk/pypistats/runs/248072462" rel="nofollow noopener">https://github.com/hugovk/pypistats/runs/248072462</a></p>
<p>Thanks!</p>
